daemon/containerd/soft_delete.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const imageNameDanglingPrefix = "moby-dangling@"
```

Comment:
For a followup: we should find a new home for this and use the same const in the builder wrapper too

Reply:
- created : https://github.com/moby/moby/issues/46613